app/src/main/java/com/vestrel00/daggerbutterknifemvp/ui/example_3/child_fragment/view/Example3ChildFragment.java

```diff
@@ -60,16 +44,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return inflater.inflate(R.layout.example_3_child_fragment, container, false);
     }
 
-    @OnClick(R.id.do_something)
-    void onDoSomethingClicked() {
-        String something = singletonUtil.doSomething();
-        something += "\n" + perActivityUtil.doSomething();
-        something += "\n" + perFragmentUtil.doSomething();
-        something += "\n" + perChildFragmentUtil.doSomething();
-        showSomething(something);
+    @Override
+    public void showSomething(String something) {
+        childFragmentManager.hashCode();
```

Comment:
This should not be here...

Reply:
Removed in: https://github.com/vestrel00/android-dagger-butterknife-mvp/commit/ea24fe34ce03d0d43eb0fce54390d443d9b0d997